pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.js

```diff
@@ -108,8 +122,20 @@ router.get('/', function (req, res, next) {
             }
           }
         });
-
-        res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+        if (res.locals.assessment.group_work) {
+          sqldb.query(sql.get_group_info, params, function (err, result) {
```

Comment:
Could this use the shared `getConfigInfo` function instead?

Reply:
This refactor happened in [4baf174](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/4baf1740683c146a1b89ad57c707d5f3942bc555) (apologies, a lot more got thrown in this commit than would have been ideal)